src/Text/Pandoc/Readers/JATS.hs

```diff
@@ -138,18 +138,16 @@ trimNl = T.dropAround (== '\n')
 
 -- function that is used by both graphic (in parseBlock)
 -- and inline-graphic (in parseInline)
-getGraphic :: PandocMonad m
-           => Maybe (Inlines, Text) -> Element -> JATS m Inlines
```

Comment:
Noticed this function was only ever called with Nothing as first argument, which means it never actually uses it. Got rid of it.